doc/getting-started.md

```diff
@@ -370,3 +371,10 @@ print(water.get_positions())
 ```
 [DP]:https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.143001
 [DP-SE]:https://dl.acm.org/doi/10.5555/3327345.3327356
+
+## Important hint
```

Comment:
I would rename the title as "Know limitations"

Reply:
solved